docs/photoniq/fingerprinting/get-started-fingerprinting.md

```diff
@@ -52,9 +62,11 @@ In this example, the JavaScript is loaded at page load time, and the function to
 </body>
 ```
 
-## Fingerprint on Button Click
+</TabItem>
 
-In this example, the JavaScript is initially loaded at page load time, and a function is created that is tied to a button. Once the button is clicked, a call is made to the fingerprint server, which returns the visitorId. The visitorId is returned in the response header (x-photoniq-vid) and the JSON body’s response (visitorId).
+<TabItem value="button click" label="Activate on button click">
+
+With this script, Javascript loading occurs at page load time, and triggers a button to create a function. Clicking this button makes a call to the fingerprint server, which returns the `visitorId` in the response header (x-photoniq-vid) and the JSON body’s response (visitorId).
```

Comment:
this does not seem to be right - `triggers a button to create a function`

Reply:
Okay, does ' With this script, when Javascript loading occurs at page load time, it creates a function tied to a button'? work better